src/Vault.sol

```diff
@@ -516,13 +527,20 @@ contract Vault is IERC4626, ERC20Permit, ILiquidationSource, Ownable {
     uint8 _v,
     bytes32 _r,
     bytes32 _s
-  ) external returns (uint256) {
+  ) external onlyVaultCollateralized returns (uint256) {
```

Comment:
(same as above) This function technically already checks collateralization since it's checked on this line of `_depositAssets`: https://github.com/GenerationSoftware/pt-v5-vault/pull/51/commits/8e0d39febf72dc79c2c8a51f969646a5c2ff1918#diff-97c974f5c3c03a0cfcbc52a5b8b9ae2196d535754ff2034e2903de1fec23508aR1277

Reply:
Fixed in this commit: https://github.com/GenerationSoftware/pt-v5-vault/pull/51/commits/dda6ac44c4f7787dce9c1e75475b0f21a51e2cb0